setup.py

```diff
@@ -9,49 +9,40 @@
 with open('plumpy/version.py') as f:
     exec(f.read(), about)
 
-setup(name='plumpy',
-      version=about['__version__'],
-      description='A python workflow library',
-      long_description=open('README.rst').read(),
-      url='https://github.com/muhrin/plumpy.git',
-      author='Martin Uhrin',
-      author_email='martin.uhrin@gmail.com',
-      license=__license__,
-      classifiers=[
-          'License :: OSI Approved :: MIT License',
-          'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
-          'Programming Language :: Python :: 3.5',
-          'Programming Language :: Python :: 3.6',
-          'Programming Language :: Python :: 3.7',
-          'Programming Language :: Python :: 3.8',
-      ],
-      keywords='workflow multithreaded rabbitmq',
-      install_requires=[
-          'frozendict',
-          'tornado>=4.1, <5.0',
-          'pyyaml~=5.1.2',
-          'pika>=1.0.0',
-          'kiwipy[rmq]>=0.5.2',
-      ],
-      extras_require={
-          'dev': [
-              'pip',
-              'pytest>4',
-              'ipython>=4.0,<6.0',
-              'twine',
-              'pytest-cov',
-              'pre-commit',
-              'shortuuid',
-              'yapf',
-              'prospector',
-              'pylint<=2.3.1; python_version>="3"',
-          ],
-          'docs': [
-              'Sphinx==1.8.4',
-              'Pygments==2.3.1',
-              'docutils==0.14',
-              'sphinx-rtd-theme==0.4.3',
-          ],
-      },
-      packages=['plumpy', 'plumpy/base'],
-      test_suite='test')
+setup(
+    name='plumpy',
+    version=about['__version__'],
+    description='A python workflow library',
+    long_description=open('README.rst').read(),
+    url='https://github.com/muhrin/plumpy.git',
+    author='Martin Uhrin',
+    author_email='martin.uhrin@gmail.com',
+    license=__license__,
+    classifiers=[
+        'License :: OSI Approved :: MIT License',
+        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+    ],
+    keywords='workflow multithreaded rabbitmq',
+    install_requires=[
+        'frozendict',
+        'tornado>=4.1, <5.0',
+        'pyyaml~=5.1.2',
+        'pika>=1.0.0',
+        'kiwipy[rmq]>=0.5.2',
+    ],
+    extras_require={
+        'dev': ['pytest~=5.4', 'pre-commit~=1.20', 'prospector~=1.2', 'yapf~=0.29', 'shortuuid'],
```

Comment:
I see you've removed `twine` from here.  That was (maybe is) used as part of the `release.sh` script to push to `pypi`.  Was this intentional?

Reply:
Since this is not really needed for the CI I thought to remove it and save some install time (although it will be minimal, I admit). The one time we need to release, I anyway have twine installed pretty much, or `pip install twine` does not seem to costly. But if you prefer to keep it, I will happily reinstate it.

Reply:
I've had a few instances where (before this being part of the `dev` rule) didn't have it in my env and it would fail at that point of the release script.  That was fine because I knew what to do and just continued manually from that point after installing it but it would be suboptimal if a dev pushed a version to `master` that didn't end up on `pypi`.

If you feel that someone other than you will be releasing my recommendation would be to do one of:
1) Put a `pip install twine` in the release script
2) Put in the wiki here release instructions
3) Leave it in `setup.py`.

In any case I don't see this as needing to hold up this PR.

Reply:
I have opened #154 to deal with this once this work is done and when preparing the next release